recipes-games/pinball/pinball_0.3.20201218.bb

```diff
@@ -0,0 +1,77 @@
+SECTION = "games"
+LICENSE = "GPLv2 & CC0-1.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=94d55d512a9ba36caa9b7df079bae19f"
+LIC_FILES_CHKSUM += "file://debian/copyright;md5=ef0d1a481815636a22c1d73b1233d9c0"
+
+SRC_URI = "git://github.com/adoptware/pinball.git;protocol=https;nobranch=1"
+SRCREV ?= "fdb2fa3ebea8a1c20e1a50a54c5df292d342b531"
+
+PR ?= "r0"
+
+S = "${WORKDIR}/git"
+
+DEPENDS += " alsa-lib"
+DEPENDS += " libglu"
+DEPENDS += " libsdl"
+DEPENDS += " libsdl-image"
+DEPENDS += " libsdl-mixer"
+DEPENDS += " libtool"
+DEPENDS += " libvorbis"
+DEPENDS += " libx11"
+DEPENDS += " tiff"
+DEPENDS += " virtual/libgl"
+
+DISTRO_FEATURES_append = " opengl x11"
+DISTRO_FEATURES_remove = " wayland" 
```

Comment:
NAK: recipes must not change DISTRO_FEATURES. You may want to inherit fearures_check and add REQUIRED_DISTRO_FEATURES = "opengl x11"

Reply:
OK